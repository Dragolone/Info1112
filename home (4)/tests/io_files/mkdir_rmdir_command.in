mkdir new_directory
ls -m
rmdir new_directory
ls -m
exit
