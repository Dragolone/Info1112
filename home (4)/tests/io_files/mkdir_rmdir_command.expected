io_files, new_directory, run_tests.sh
io_files, run_tests.sh
